tests/fake_task_manager.py

```diff
@@ -31,6 +31,7 @@ def setup_task(
             "binary_image": binary_image,
             "binary_image_resolved": binary_image + "-resolved",
             "index_image": "index_image",
+            "index_image_resolved": "index_image_resolved",
```

Comment:
Could you help me in understanding if adding index_image_resolved is directly related to adding deprecation_list or was it needed to be added anyway?

Reply:
I found it was not implemented there. I don't remember if it failed without that or not.. I will try to check if it can be removed.

Reply:
removed